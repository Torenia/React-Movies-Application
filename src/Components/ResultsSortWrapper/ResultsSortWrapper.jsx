import styled from 'styled-components';

export const ResultsSortWrapper = styled.div`
  display: flex;
  text-transform: uppercase;
  color: #555;
  
  select {
    background-color: #232323;
    color: #fff;
    border: none;
    text-transform: uppercase;
    margin-left: 15px;
    outline:none;
  }
`;
