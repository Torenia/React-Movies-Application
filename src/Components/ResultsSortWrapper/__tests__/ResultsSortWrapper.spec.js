import React from 'react';

import { ResultsSortWrapper } from '../ResultsSortWrapper';

describe('<ResultsSortWrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ResultsSortWrapper />);
  });

  it('should render results sort wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
