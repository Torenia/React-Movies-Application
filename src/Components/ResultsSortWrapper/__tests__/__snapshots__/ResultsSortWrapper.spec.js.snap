// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultsSortWrapper /> should render results sort wrapper 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  text-transform: uppercase;
  color: #555;
}

.c0 select {
  background-color: #232323;
  color: #fff;
  border: none;
  text-transform: uppercase;
  margin-left: 15px;
  outline: none;
}

<div
  className="c0"
/>
`;
